index.html

```diff
@@ -93,14 +93,14 @@ <h2 class="page-header"><i class="fa fa-cubes"></i> Projects</h2>
      <ul>
      <li><a href="https://github.com/JuliaReach/LazySets.jl/">LazySets.jl</a> A Julia package for calculus with convex sets.</li>
      <li><a href="https://github.com/JuliaReach/Reachability.jl/">Reachability.jl</a> Reachability and Safety of Nondeterministic Dynamical Systems.</li>
-     <li><a href="https://github.com/JuliaReach/MathematicalSystems.jl/">MathematicalSystems.jl</a> Systems definitions in Julia.</li>
      <br>
      </ul>
 
 <p align="justify">
 The following component packages are developed and maintained by the JuliaReach org. 
 </p>
      <ul>
+     <li><a href="https://github.com/JuliaReach/MathematicalSystems.jl/">MathematicalSystems.jl</a> Systems definitions in Julia.</li>
```

Comment:
I wonder if we should remove the "in Julia" everywhere because it is quite obvious.